crafters/nlp/PDFExtractor/__init__.py

```diff
@@ -0,0 +1,39 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+from jina.executors.crafters import BaseCrafter
+
+
+class PDFTextExtractor(BaseCrafter):
+    """
+    :class:`PDFTextExtractor` Extracts text from PDF.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def post_init(self):
+        import PyPDF2
+        import io
+
+    def craft(self, uri: str, buffer: bytes, *args, **kwargs):
+        import PyPDF2
+        import io
+        text = ""
+        if buffer:
```

Comment:
Instead of using 2 variables `uri` & `buffer`, we can use just 1 and do type checking (`bytes` vs `str`)

Reply:
Driver is passing the info depending on the content I think, similar to what `ImageReader` is having.

Reply:
Oh! This is coming from the `Document` proto. Got it. Apologies!

Reply:
None apologies needed!